tools/scripts/brokenLinkChecker/brokenLinkChecker.js

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env node
+
+const blc = require('broken-link-checker');
+const https = require('https');
+const url = require('url');
+
+var whitelist = require('./whitelist.json').ignore;
+
+const slack_url =
+  // test room URL
+  "https://hooks.slack.com/services/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";
+
+  // staging room URL
+  //"https://hooks.slack.com/services/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";
+
+function numberWithCommas(x) {
+  return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+}
+
+var totalLinkCount = 0;
+var brokenLinkCount = 0;
+var whitedlistedLinkCount = 0;
+
+/* Returns true if the provided string contains one of the whitelisted strings.
+ * Note that it doesn't need to be an exact match; rather a whitelisted string
```

Comment:
wonder if it would be better if our white list was regexes so that we have flexibility around whether it needs ot be an exact match or not.